modules/users/client/controllers/settings/change-password.client.controller.js

```diff
@@ -33,11 +33,13 @@
       // If successful show success message and clear form
       $scope.$broadcast('show-errors-reset', 'vm.passwordForm');
       vm.success = true;
```

Comment:
https://github.com/meanjs/mean/blob/master/modules/core/tests/client/interceptors/auth-interceptor.client.tests.js#L26
https://github.com/meanjs/mean/blob/master/modules/core/tests/client/interceptors/auth-interceptor.client.tests.js#L49

Something similar can be done. The first links shows how to setup the Spy to watch the specific service method. You can setup a Spy for each Notification method that you expect to be called; in this case, `Notification.error()` & `Notification.success()`.